vck/src/commonMain/kotlin/at/asitplus/wallet/lib/jws/JwsService.kt

```diff
@@ -48,6 +54,7 @@ interface JwsService {
      */
     val keyMaterial: KeyMaterial
 
+    @Deprecated("Use SignJwtFun instead")
```

Comment:
add `replaceWith=…` to the annotation

Reply:
But it's not really an exact replacement, so I've not included that